development/tools/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:alpine AS builder
+
+WORKDIR /go/src/github.com/kyma-project/test-infra
+COPY . development/tools
+
+RUN apk --update --no-cache add bash upx dep git && \
+    development/tools/build-cleaners.sh
+
+FROM alpine:latest
+
+RUN apk --update --no-cache add ca-certificates bash
+COPY --from=builder /go/src/github.com/kyma-project/test-infra/development/tools/bin /tools
+ENV PATH $PATH:/tools
```

Comment:
we don't need to add it to the path, you could add `WORKDIR /tools` and then in the jobs itself we can either use ENTRYPOINT that just points to the executable `["./tool1"]` or run the command will point to the tool.

Reply:
Maybe a nice gimmick would be to add `ENTRYPOINT ["echo", "please select a tool from the list", ";", "ls -alh"]` or something like that, so when you run it without passing a tool name it will print you the available tools.

Reply:
I changed it in recent commits.